crates/base/src/server.rs

```diff
@@ -87,24 +90,43 @@ impl Server {
         ip: &str,
         port: u16,
         main_service_path: String,
+        maybe_events_service_path: Option<String>,
         import_map_path: Option<String>,
         no_module_cache: bool,
         callback_tx: Option<Sender<ServerCodes>>,
     ) -> Result<Self, Error> {
+        let mut worker_event_sender: Option<mpsc::UnboundedSender<WorkerEvents>> = None;
+
+        // Creates Event Worker
+        if let Some(events_service_path) = maybe_events_service_path {
+            let events_path = Path::new(&events_service_path);
+            let events_path_buf = events_path.to_path_buf();
+
+            let event_worker =
+                create_event_worker(events_path_buf, import_map_path.clone(), no_module_cache)
```

Comment:
It creates an event manager for all the the user function pool. So all functions will report to this instance